pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>lfp-appeals-api</artifactId>
+    <version>0.0.1</version>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <spring.boot.version>2.2.4.RELEASE</spring.boot.version>
```

Comment:
I have never done what I am about to propose next but maybe we could use the following pattern?

`<version.spring.boot></version.spring.boot>`

and 

`<version.lombok></version.lombok>`

and 

`<version.springdoc.openapi></version.springdoc.openapi>`

Pattern I propose is more or less:

`version.[vendor].[product]` where version is fixed group to group all version properties together.

Reply:
sure :) looks cleaner!